apps/server/src/shared/domain/service/lesson-copy.service.ts

```diff
@@ -34,13 +35,15 @@ export class LessonCopyService {
 	) {}
 
 	async copyLesson(params: LessonCopyParams): Promise<CopyStatus> {
-		const { copiedContent, contentStatus } = await this.copyLessonContent(params.originalLesson.contents || [], params);
+		const { copiedContent, contentStatus } = await this.copyLessonContent(params.originalLesson.contents, params);
```

Comment:
this extra case was the one we did not cover in tests. I asked myself if we really need that extra case or if that is covered by the entity providing 
`@Property()
	contents: IComponentProperties[] | [];`
would be enough. what do you think @virgilchiriac

Reply:
yes, seems to work ok, ts not complaining. In any case, in the propriety and if needed in the entity constructor before here